edx/edx-ora2
# [BD-5] BB-2504: Add grade status summary page. [WIP]

Add a status message on the “Your grade” step explaining how the final grade is calculated/determined.

**JIRA tickets**: 
* https://openedx.atlassian.net/browse/TNL-7274
* https://openedx.atlassian.net/browse/OSPR-4624

**Dependencies**: None

**Screenshots**: Always include screenshots if there is any change to the UI.

**Merge deadline**: None

**Testing instructions**:

1. Create a sample course.
2. Create ORA (Open Response Assessment) one for each assessment type e.g **self**, **peer**, **staff**, **self** + **peer**, ....., etc.
3. Create student and enroll them into the created course.
4. Get them to attempt the ORA assessments and get them to grading page.
5. Check the output HTML whether the grade explanation message appears or not.

**Author notes and concerns**:

1. There is some strings missing and for them I have added pseudo strings. I will replace them with actual ones once I get them from upstream.

**Reviewers**
